### PR TITLE
Ask va prototype

### PR DESCRIPTION
**Pull request by joehall-tw**
## Summary

- This PR adds two placeholder pages to the vets-website repo for a new application.
- The content-build repo has. a parallel change to add the same app to `registry.json`, with vagovprod set to false.
- This is the first in a series of edits to get the app created in stages: placeholder, mockup, and initial application
- **NOTE:** the purpose of this application is to move the askva.va.gov portal to the va.gov website. There were naming conflicts for the desired name (ask-va) so we called this app "ask-va-too". This is a temporary name until we plan our strategy to replace the existing references to the old pages.
- We are the Ask VA Integration team, and we would like to own the maintenance of this new application.
- There are no flippers at this time since we assume the `registry.json` entry in the `content-build` repo has a vagovprod value of false. Also, no existing pages reference this new src/application. If we need a toggle, let us know and we'll add a flipper to disable the application in production.

## Related issue(s)

- There are no related issues in the va.gov-team repo
- our issue that drove this work is documented here: https://github.com/department-of-veterans-affairs/ask-va/issues/2
- our team is referenced in the va.gov-team under `products/ask-va`

## Testing done

- This is a set of placeholder pages. There are two tests (one for each route) that verify that the pages were rendered.
- Visit the `/contact-us/ask-va-too` page to see the placeholder text.
- Both tests are passing locally. This indicates that the two pages (routes to two controls) are reachable and render the expected text.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

Our placeholder pages use no va.gov controls, but the next update (the mock pages) will.

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

- except for the sass reference in `babel.config.json`, no code outside of the `/src/applications/ask-va` folder was modified.

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

This is our first push to get the scaffolding in place to develop our application for the va.gov site. We followed the instructions provided. Please let us know if we need to make other edits. 

Any feedback, advice, or caveats are welcome.

### CODE REVIEW

File: src/applications/ask-va/components/page/AvaFormPage.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @department-of-veterans-affairs/prefer-telephone-component */
```
[the remark this answers is not in the document]
Comment (joehall-tw) [reply src/applications/ask-va/components/page/AvaFormPage.jsx]: Removed unneeded lint flag.

File: src/applications/ask-va/components/page/LandingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @department-of-veterans-affairs/prefer-telephone-component */
+/* eslint-disable @department-of-veterans-affairs/telephone-contact-digits */
```
[the remark this answers is not in the document]
Comment (joehall-tw) [reply src/applications/ask-va/components/page/LandingPage.jsx]: Removed unneeded lint flag.

File: src/applications/ask-va/components/page/LandingPage.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @department-of-veterans-affairs/prefer-telephone-component */
```
[the remark this answers is not in the document]
Comment (joehall-tw) [reply src/applications/ask-va/components/page/LandingPage.jsx]: Removed unneeded lint flag.

**Review by ediiotero — APPROVED** [review]
looks good

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**